src/types/KeyEvent.cpp

```diff
@@ -68,7 +68,7 @@ void KeyEvent::ActivateModifierKey(const ModifierKeyState modifierKey) noexcept
     WI_SetAllFlags(_activeModifierKeys, bitFlag);
 }
 
-bool KeyEvent::DoActiveModifierKeysMatch(const std::unordered_set<ModifierKeyState>& consoleModifiers) const noexcept
+bool KeyEvent::DoActiveModifierKeysMatch(const std::unordered_set<ModifierKeyState>& consoleModifiers) const

```

Comment:
Where can this except?

Reply:
Apparently on `std::unordered_set<T>`, it can except on...
`_Unchecked_begin()`
`_Unchecked_end()`
`operator!=()`
`operator*()`
`operator++()`

So basically the entire foreach loop process.